docs/developer/dependencies.md

```diff
@@ -59,16 +59,18 @@ sudo installer -pkg /Library/Developer/CommandLineTools/Packages/macOS_SDK_heade
 
 ## Haskell Stack
 
-When you're done, ensure `stack --version` is >= 1.6.5
+Please refer to [Stack's installation instructions](https://docs.haskellstack.org/en/stable/README/#how-to-install).
+
+When you're done, ensure `stack --version` is recent, ideally the same as `STACK_ALPINE_VERSION` in [`build/alpine/Dockerfile.prebuilder`](../../build/alpine/Dockerfile.prebuilder).
```

Comment:
Do we want the Dockerfile to be the main source of truth for the Stack version? We could also add the version we're using here (of course with the cost that we always need to remember to update both places).

Reply:
I think it's okay to have the docker image be the "source of truth" so we don't need to change the version in multiple places.